union-ai-admin/aws/gen/unionai-provisioner-role.template.yaml

```diff
@@ -35,12 +35,6 @@ Resources:
               - !Sub 'arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:opta-*'
               - !Sub 'arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group::log-stream*'
               - !Sub 'arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/eks/opta-*:*'
-          - Action:
```

Comment:
These cloudfront permissions were removed in a previous change, it seems we forgot to check in the generated templates.